src/app/Views/login.view/login.view.html

```diff
@@ -0,0 +1,33 @@
+<div class="mdc-card mdc-card--outlined">
+
+  <form class="login-form">
```

Comment:
this is a login view that also holds an actual implementation of the form. `Views` should render specific components.
We should separate this  part in a `Core` component and use it here